src/test/java/com/azure/spring/data/cosmos/exception/CosmosNonUniqueExceptionTest.java

```diff
@@ -0,0 +1,14 @@
+package com.azure.spring.data.cosmos.exception;
+
```

Comment:
What is the purpose of this file?

Reply:
This test class was accidentally added and I will remove it